src/Tribe/Editor/Attendee_Registration.php

```diff
@@ -35,7 +35,7 @@ public function hook() {
 	 */
 	public function filter_admin_body_class( string $classes ) {
 		// if not on attendee registration page
-		if ( tribe_get_request_var( 'page', '' ) ) {
+		if ( ! ( tribe_get_request_var( 'page', '' ) === 'attendee-registration' ) ) {
```

Comment:
This is now changed as of the latest work that @Camwyn did, is that correct Stephen?

Reply:
```suggestion
		$ar_page_slug = tribe( 'tickets.attendee_registration' )->get_slug();
		
		if ( tribe_get_request_var( 'page', '' ) !== $ar_page_slug ) {
```

Reply:
Please retest with the tweak above